### PR TITLE
TM-1117 api docs

### PR DESCRIPTION
**Pull request by kallosz**

### CODE REVIEW

**Comment by ArekZc** [comment]
Every route missing description about what represent. For example. /containers - Represents a [Container](https://7tag.org/docs/containers/). The /containers node returns list of containers.

**Comment by ArekZc** [comment]
Triggers routes missing property description

**Comment by ArekZc** [comment]
website route missing property description

**Comment by ArekZc** [comment]
Missing descriptions for property in oauth token
zrzut ekranu 2016-01-22 o 09 43 10

**Comment by ArekZc** [comment]
Missing description for collection properties
zrzut ekranu 2016-01-22 o 09 44 42

**Comment by ArekZc** [comment]
Build failed

**Comment by ArekZc** [comment]
missing route for variables, should be method post to add variable
zrzut ekranu 2016-01-26 o 11 16 26

**Comment by ArekZc** [comment]
variables put route missing description for property
zrzut ekranu 2016-01-26 o 11 16 40

**Comment by MarcinMat** [comment]
**/oauth/v2/token** requires also 'username' and 'password' to work

**Comment by MarcinMat** [comment]
Missing routes - I don't know which ones should be included, so I'm going to list them all:

get_condition_types                                          GET    ANY    ANY  /api/containers/{id}/conditions
put_containers_permissions                                   PUT    ANY    ANY  /api/containers/{id}/permissions
get_containers_permissions                                   GET    ANY    ANY  /api/containers/{id}/permissions
post_containers                                              POST   ANY    ANY  /api/containers

get_variables_types                                          GET    ANY    ANY  /api/variable-types

others_settings                                              PUT    ANY    ANY  /api/users/me/others-settings
change_password                                              POST   ANY    ANY  /api/users/me/change-password
reset_password_users                                         POST   ANY    ANY  /api/users/{id}/reset-password
get_users_me                                                 GET    ANY    ANY  /api/users/me
put_users_me                                                 PUT    ANY    ANY  /api/users/me
delete_users                                                 DELETE ANY    ANY  /api/users/{id}
get_users                                                    GET    ANY    ANY  /api/users
post_users                                                   POST   ANY    ANY  /api/users
put_users                                                    PUT    ANY    ANY  /api/users/{id}
get_users_by_id                                              GET    ANY    ANY  /api/users/{id}
logout                                                       GET    ANY    ANY  /api/users/me/logout

reset_password_token                                         POST   ANY    ANY  /api/reset-password/token/{token}
reset_password_request                                       POST   ANY    ANY  /api/reset-password/request

get_integration                                              GET    ANY    ANY  /api/integration/{id}
post_integrationseventag_api_integration_post                POST   ANY    ANY  /api/integration
put_integrationseventag_api_integration_put                  PUT    ANY    ANY  /api/integration/{id}
delete_integrationseventag_api_integration_delete            DELETE ANY    ANY  /api/integration/{id}
get_integrations                                             GET    ANY    ANY  /api/integration

fos_reset_password                                           ANY    ANY    ANY  /reset/{token}
fos_oauth_server_authorize                                   POST   ANY    ANY  /api/oauth/v2/auth
index                                                        GET    ANY    ANY  /api/translations
get                                                          GET    ANY    ANY  /api/translations/{lang}

File: src/SevenTag/Api/TagBundle/Form/TagType.php [hunk 1]
```diff
@@ -50,20 +50,21 @@ public function __construct(HolderInterface $templateHolder)
...
                 'accessible',
                 [
-                    'class' => 'SevenTag\Api\TriggerBundle\Entity\Trigger'
+                    'class' => 'SevenTag\Api\TriggerBundle\Entity\Trigger',
+                    'description' => 'Trigger assigned to tag'
                 ]
             )
-            ->add('priority', 'integer')
+            ->add('priority', 'integer', ['description' => 'Tag priority'])
             ->add('template', 'text');
```
Comment (MarcinMat) [inline src/SevenTag/Api/TagBundle/Form/TagType.php]: No description here. Also no info about the difference for code and template. We use template if tag is one of the predefined ones f.e. marketo, while code is for custom snippet.

**Pull request merged**